pkg/types/taskPayload.go

```diff
@@ -26,8 +26,9 @@ type FriendNotificationTaskPayload struct {
 
 // A list of task types.
 const (
-	TypePostNotification   = "post:deliver"
-	TypeChatNotification   = "chat:deliver"
-	TypeFriendNotification = "friend:deliver"
-	TypeSystemNotification = "system:deliver"
+	TypePostNotification      = "post:deliver"
+	TypeChatNotification      = "chat:deliver"
+	TypeFriendNotification    = "friend:deliver"
+	TypeApprovedFriendRequest = "approvedFriendRequest:deliver"
```

Comment:
Maybe we call it TypeApprovedFriend

And have the task string to be same? 

Thoughts?